lib/acts-as-taggable-on.rb

```diff
@@ -1,6 +1,7 @@
 require 'active_record'
 require 'active_record/version'
 require 'active_support/core_ext/module'
+require 'active_support/core_ext/string/starts_ends_with'
```

Comment:
why is this needed?

Reply:
Replacing `ends_with?` with `end_with?` would had also been an alternative, that require was to bring in that alias.